src/llmq/quorums_chainlocks.cpp

```diff
@@ -403,23 +384,30 @@ bool CChainLocksHandler::IsTxSafeForMining(const uint256& txid)
 }
 
 // WARNING: cs_main and cs should not be held!
+// This should also not be called from validation signals, as this might result in recursive calls
 void CChainLocksHandler::EnforceBestChainLock()
 {
     CChainLockSig clsig;
     const CBlockIndex* pindex;
+    const CBlockIndex* bestChainLockBlockIndex;
```

Comment:
I would prefer a name that would not cause a confusion with a class member, smth like `pindexBestChainLock` or `currentBestChainLockBlockIndex` or whatever.

Reply:
Renamed to currentBestChainLockBlockIndex